tests/test_yaml_schema_and_regex.py

```
@@ -0,0 +1,199 @@
+
+
+def load_yaml(file_path):
+    with open(file_path, "r") as f:
+        return yaml.safe_load(f)
+
+
+def is_valid_regex(regex: str) -> bool:
+    try:
+        pattern = re.compile(regex)
+        re.findall(pattern, DUMMY_TEST_STRING)
+        return True
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)